### DISCUSSION TITLE
Share Your Creations!

### DISCUSSION THREAD

**Discussion by tjtanjin** [tjtanjin/react-chatbotify#11] [Show and tell]
## 👋 Hi there!
Do you have an interesting chatbot to share? Perhaps you've done some cool customizations to tailor the chatbot to your project(s)? Well, we're excited to see the amazing projects you've built using  [React ChatBotify](https://react-chatbotify.tjtanjin.com) ! Whether it's a sleek web application, a creative portfolio, or a unique project, we want to [showcase](https://github.com/tjtanjin/react-chatbotify/blob/main/SHOWCASES.md) the incredible work of our community.

### Submission Guidelines 📝
To ensure the quality and readiness of the projects shared, please adhere to the following guidelines:

- **Production-Ready:** Your website must be in a production-ready state, not in the testing phase. We want to highlight projects that are ready for public use.

- **Submission Template:** Use the following template to submit your project. Copy and paste this section with your information filled in:

    - **Website Link:** [Your Project's Live URL]
    - **Repository Link (Optional):** [Link to Your Project's Repository]
    - **Project Title (≤50 characters):** [Your Project's Title]
    - **Project Description (≤100 characters):** [Briefly describe your project in one sentence]
    - **Project Category:** [Open Source | Portfolio | Education | Entertainment | Healthcare | Others]
    
- **Images**: Optionally, leave a stronger impression by including an image of your website showing the chatbot.

### Example Submission 🌐

```
Website Link: https://taverncraft.tk
Repository Link (Optional): -
Project Title (≤50 characters): TavernCraft Website
Project Description (≤100 characters): An informative website for TavernCraft, a free-to-play minecraft server!
Project Category: Entertainment
```
### Notes 📌
- Submissions will be updated within 1-2 weeks (feel free to give a gentle reminder if there are no updates after 2 weeks)
- This is not the place for support, feel free to create a new discussion, raise an issue or reach out on [discord](https://discord.gg/6R4DK4G5Zh) if you need help
- You may choose more than one **Project Category** if applicable, in fact we encourage choosing all categories applicable to your project
- If you need an update to your entry, simply create a new submission but do help us by making a note that it is an update
- If any of the content shared is deemed inappropriate, they may be rejected/removed
- If you have multiple projects, keep them as separate submissions

**Comment by tjtanjin** [comment]
![image](https://github.com/tjtanjin/react-chatbotify/assets/43908963/751d8a92-c807-45b2-a39d-659937471a7d)

Website Link: https://react-chatbotify.tjtanjin.com
Repository Link (Optional): https://github.com/tjtanjin/react-chatbotify-docs
Project Title (≤50 characters): React ChatBotify Documentation Website
Project Description (≤100 characters): A documentation website for the React ChatBotify library
Project Category: Open Source

**Comment by tjtanjin** [comment]
![image](https://github.com/tjtanjin/react-chatbotify/assets/43908963/91b1af71-e6f2-458e-b21f-44cdff9d85d8)

Website Link: https://taverncraft.tk
Repository Link (Optional): -
Project Title (≤50 characters): TavernCraft Website
Project Description (≤100 characters): An informative website for TavernCraft, a free-to-play minecraft server!
Project Category: Entertainment

**Comment by Andres2D** [comment]
![Screenshot 2024-05-03 at 3 23 27 AM](https://github.com/tjtanjin/react-chatbotify/assets/37199673/1956cad0-1d6e-4f3b-a513-d398ce227e44)


Website Link: https://andres-alcaraz.netlify.app/
Repository Link: https://github.com/Andres2D 
Project Title (≤50 characters): Web development portfolio
Project Description (≤100 characters): A portfolio showcasing all my projects, experience and skills.
Project Category: Portfolio.

**Comment by AKSinsane1997** [comment]
hi @tjtanjin , i want to change the language attribute in voice option like"hi-IN" , but there is no property in the documentation. kindly update on this please.
![Screenshot (50)](https://github.com/tjtanjin/react-chatbotify/assets/81317739/a1c6340c-36a1-476c-b486-ca4960f3d28a)

**Reply by tjtanjin** [reply]
Hey @AKSinsane1997! I've replied to your question [here](https://github.com/tjtanjin/react-chatbotify/discussions/7#discussioncomment-9716730). Separately if you'd like to share your creations, kindly check the submission guidelines for the template. Thanks!
